shared/charts/backup-exporter/templates/configmap.yaml

```diff
@@ -14,9 +14,9 @@ data:
     gcloud config set account {{ .Values.serviceAccountName }}
     SNAPSHOTS=$(for i in `seq 0 $(({{ .Values.replicaCount }}-1))`; do gcloud compute snapshots list --sort-by=~creationTimestamp,STATUS --limit=1 --format="value[separator=';'](name,status)" --filter="name~'pv-database-storage-couchdb-couchdb-$i-*'" | cut -f1 -d\; ; done)
     DESTINATION_BUCKET={{ .Values.destinationBucket }}
-    TIMESTAMP=$(date +%s)
+    TIMESTAMP=$(date +%F_%H%M%S)
```

Comment:
What timezone does this use? Maybe it would be better to specify?

Reply:
Fixed in https://github.com/gpii-ops/gpii-infra/pull/400/commits/f808e1011486411176e9500a52f3455aa8b41441